File: src/main/scala/com/blackwing/easyExploration/blocks/Blocks.scala
package com.blackwing.easyExploration
import com.blackwing.easyExploration.blocks.RubyBlock
import com.blackwing.easyExploration.blocks.RubyOre
import net.minecraft.block.Block
import net.minecraft.block.material.Material


package object Blocks {

	//Blocks
	val RUBY_BLOCK = new RubyBlock("ruby_block", Material.IRON)
	val RUBY_ORE = new RubyOre("ruby_ore", Material.ROCK)
}

File: src/main/java/com/blackwing/easyExploration/blocks/Blocks.java
package com.blackwing.easyExploration.blocks;

import net.minecraft.block.Block;

public class Blocks {
    private static Blocks ourInstance = new Blocks();

    public static Blocks getInstance() {
        return ourInstance;
    }

    private Blocks() {
    }

    public static final Block DeathChest = new DeathChest("death_chest");
}
